sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Constants.java

```diff
@@ -198,12 +198,15 @@
         public static final String LOGGING_HANDLER_NAME = "logging-handler";
 
         // encryption
-        public static final String WrappedDataEncryptionKey = "wrappedDataEncryptionKey";
-        public static final String  EncryptionAlgorithmId = "encryptionAlgorithmId";
-        public static final String  KeyWrapMetadata = "keyWrapMetadata";
+        public static final String ENCRYPTION_ALGORITHM_ID = "encryptionAlgorithmId";
+        public static final String KEY_WRAP_METADATA = "keyWrapMetadata";
+        public static final String WRAPPED_DATA_ENCRYPTION_KEY = "wrappedDataEncryptionKey";
+        public static final String CLIENT_ENCRYPTION_POLICY = "clientEncryptionPolicy";
         public static final String  KeyWrapMetadataType = "type";
         public static final String  KeyWrapMetadataValue = "value";
```

Comment:
>yWr [](start = 38, length = 3)

There should be a KeyWrapMetadataName as well

Reply:
we are not using these constants , instead using string directly on EncryptionKeyWrapMetadata